src/widgets/histogram/content-view.js

```diff
@@ -49,6 +49,7 @@ module.exports = cdb.core.View.extend({
 
     this._renderMiniChart();
     this._renderMainChart();
+    this._renderAllValues();
```

Comment:
But the thing is if collapsed, we don't render the stats. If we uncollapse it, we render it back. But the values are set only on model change. We need to repaint them.